alpha/veneer/README.md

```diff
@@ -0,0 +1,22 @@
+
+## Veneers 
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions.  FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/).  Unfortunately, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.  The term is a bit of a misnomer, because a `veneer` isn’t pretending to be something else.
+
+There are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
+
```

Comment:
Maybe add here that opm includes to example veneers and a short sentence describing them, to further solidify the explanation above. Then go on to say that the list of veneers can be expanded by community contribution.